addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -46,7 +46,7 @@
     'AT': {'9915': 'vat'},
     'AU': {'0151': 'vat'},
     'BA': {'9924': 'vat'},
-    'BE': {'9925': 'vat'},
+    'BE': {'0208': 'company_registry'},
```

Comment:
9925
Belgium VAT number

```suggestion
    'BE': {'9925': 'vat', '0208': 'company_registry'},
```
Vat must remain in first position.
Since 0208 is a generic code, we don't add it to others country?

Reply:
according to BOSA, we should never use 9925 if we can. We want to ensure the 0208 code is selected for Belgian companies and that the CBE code accompanies this EAS code. They say that this is because not all Belgian companies have vat